addons/mail/controllers/webclient.py

```diff
@@ -37,16 +39,30 @@ def _process_request(self, **kwargs):
         return store.get_result()
 
     def _process_request_for_all(self, store, **kwargs):
-        if kwargs.get("init_messaging"):
-            if not request.env.user._is_public():
-                user = request.env.user.sudo(False)
-                user._init_messaging(store)
+        if "init_messaging" not in kwargs:
+            return
```

Comment:
There might/will be more options inside this method, a quick return is not something we'll have in future PR.

Reply:
reverted